File: test_absences.js
import { createClient } from '@supabase/supabase-js';
import dotenv from 'dotenv';
dotenv.config({ path: '.env.local' });

const supabaseUrl = process.env.VITE_SUPABASE_URL;
const supabaseKey = process.env.VITE_SUPABASE_ANON_KEY;
const supabase = createClient(supabaseUrl, supabaseKey);

async function test() {
  const { data, error } = await supabase.from('teacher_attendance').select('*').eq('date', '2026-06-04');
  console.log("Error:", error);
  console.log("Data length:", data?.length);
  console.log("Data:", JSON.stringify(data, null, 2));
}

test();

File: test_absences.py
import os
import requests
from dotenv import load_dotenv

load_dotenv('.env.local')

supabase_url = os.environ.get('VITE_SUPABASE_URL')
supabase_key = os.environ.get('VITE_SUPABASE_ANON_KEY')

headers = {
    'apikey': supabase_key,
    'Authorization': f'Bearer {supabase_key}'
}

response = requests.get(
    f"{supabase_url}/rest/v1/teacher_attendance?date=eq.2026-06-04",
    headers=headers
)

print(response.status_code)
print(response.json())
